hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/filtermanagement/CustomTargetBeanQuery.java

```diff
@@ -45,7 +45,7 @@
 public class CustomTargetBeanQuery extends AbstractBeanQuery<ProxyTarget> {
 
     private static final long serialVersionUID = 6490445732785388071L;
-    private Sort sort = new Sort(Direction.DESC, "createdAt");
+    private Sort sort = new Sort(Direction.ASC, "id");
```

Comment:
I changed the order of the FilterManagementTable from latest to bottom. Before it was latest top which was the only table which behaved like this. So for consistent I changed it as well.